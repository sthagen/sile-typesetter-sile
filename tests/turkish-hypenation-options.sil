\begin[papersize=a7]{document}
\use[module=packages.retrograde,target=v0.15.0]
\nofolios
\neverindent
\font[size=32pt,language=tr]
\set[parameter=document.baselineskip,value=32]

% A non-apostrophe break
İstanbul'dansınız.

% An apostrophe break point
\kern[width=72pt]
İstanbul'dansınız.

% An apostrophe break point with curly quotes
\kern[width=72pt]
İstanbul’dansınız.

\set[parameter=languages.tr.replaceApostropheAtHyphenation,value=true]
\kern[width=72pt]
İstanbul'dansınız.

\end{document}
