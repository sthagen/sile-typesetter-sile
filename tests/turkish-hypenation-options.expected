Set paper size 	209.7637818	297.6377985
Begin page
Mx 	10.4882
My 	40.4444
Set font 	Gentium Plus;32;400;;normal;;;LTR
T	997 86 w=21.8906	(İs)
Mx 	32.3788
T	87 68 81 w=43.3594	(tan)
Mx 	75.7382
T	69 88 79 w=41.8750	(bul)
Mx 	117.6132
T	10 w=8.0000	(')
Mx 	125.6132
T	71 68 81 w=48.9844	(dan)
Mx 	174.5976
T	86 215 w=21.0313	(sı)
Mx 	195.6288
T	16 w=10.7813	(-)
Mx 	10.4882
My 	72.4444
T	81 215 93 w=39.9063	(nız)
Mx 	50.3944
T	17 w=7.3281	(.)
Mx 	88.5955
My 	104.4444
T	997 86 w=21.8906	(İs)
Mx 	110.4861
T	87 68 81 w=43.3594	(tan)
Mx 	153.8455
T	69 88 79 w=41.8750	(bul)
Mx 	195.7205
T	10 w=8.0000	(')
Mx 	10.4882
My 	136.4444
T	71 68 81 w=48.9844	(dan)
Mx 	59.4726
T	86 215 w=21.0313	(sı)
Mx 	80.5038
T	81 215 93 w=39.9063	(nız)
Mx 	120.4101
T	17 w=7.3281	(.)
Mx 	88.5955
My 	168.4444
T	997 86 w=21.8906	(İs)
Mx 	110.4861
T	87 68 81 w=43.3594	(tan)
Mx 	153.8455
T	69 88 79 w=41.8750	(bul)
Mx 	195.7205
T	183 w=7.3281	(’)
Mx 	10.4882
My 	200.4444
T	71 68 81 w=48.9844	(dan)
Mx 	59.4726
T	86 215 w=21.0313	(sı)
Mx 	80.5038
T	81 215 93 w=39.9063	(nız)
Mx 	120.4101
T	17 w=7.3281	(.)
Mx 	88.5955
My 	232.4444
T	997 86 w=21.8906	(İs)
Mx 	110.4861
T	87 68 81 w=43.3594	(tan)
Mx 	153.8455
T	69 88 79 w=41.8750	(bul)
Mx 	195.7205
T	16 w=10.7813	(-)
Mx 	10.4882
My 	264.4444
T	71 68 81 w=48.9844	(dan)
Mx 	59.4726
T	86 215 w=21.0313	(sı)
Mx 	80.5038
T	81 215 93 w=39.9063	(nız)
Mx 	120.4101
T	17 w=7.3281	(.)
End page
Finish
